concent-vm/vars.yml

```diff
@@ -1,2 +1,5 @@
-concent_deployment_dir: /home/builder/concent-deployment
-concent_dir:            /home/vagrant/concent
+concent_deployment_dir:      /home/builder/concent-deployment
+concent_dir:                 /home/vagrant/concent
+golem_dir:                   /home/vagrant/golem
+golem_hyperdrive_repository: https://github.com/mfranciszkiewicz/golem-hyperdrive/releases/download/v0.2.6/hyperg_0.2.6_linux-x64.tar.gz
```

Comment:
https://github.com/mfranciszkiewicz/golem-hyperdrive/releases/download/v0.2.6/hyperg_0.2.6_linux-x64.tar.gz is not a repository. It's a release package URL.

Please simply set `golem_hyperdrive_version` to `0.2.6` and put the rest of the path in the playbook. Or at least in a separate variable:

``` yam
hyperg_version: "0.2.6"
hyperg_url: https://github.com/mfranciszkiewicz/golem-hyperdrive/releases/download/v{{ hyperg_version }}/hyperg_{{ hyperg_version }}_linux-x64.tar.gz
```